code/datums/mind.dm

```diff
@@ -395,6 +395,8 @@
 		var/obj_count = 1
 		for(var/datum/objective/objective in antag_objectives)
 			output += "<br><B>Objective #[obj_count++]</B>: [objective.explanation_text]"
+			if (objective.name == "gimmick")
```

Comment:
Rather than check for a specific string, make a var called optional and check for that, so all optional objectives can be given it even if not of a specific subtype. If that wasnt an option, better coding practice would be to check type rather than string.

Reply:
resolved